lib/c/build/bin/cc-link

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
I was wondering what you were going to call this file. In my mind it's `llvm-ld`, but you've probably thought more about this now than I have.

Reply:
To be honest, my naming of all these things is actually not exactly consistent. I couldn't quite figure out how to make it so but I will have another quick look and see if your suggestion helps.

Reply:
I chose `libc-link` which finally makes things consistent at least. I think my original choice of using `ld` was also a bit silly and misleading so I have killed that pattern.